src/atomate2/vasp/flows/magnetism.py

```diff
@@ -0,0 +1,100 @@
+"""VASP-specific flows for calculating magnetic orderings and other magnetism-related tasks."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Literal, Sequence
+
+from atomate2.common.flows.magnetism import (
+    MagneticOrderingsMaker as MagneticOrderingsMakerBase,
+)
+from atomate2.vasp.jobs.core import RelaxMaker, StaticMaker
+
+if TYPE_CHECKING:
+    from pymatgen.core import Element
+
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+__all__ = ["MagneticOrderingsMaker"]
+
+
+@dataclass
+class MagneticOrderingsMaker(MagneticOrderingsMakerBase):
+    """
+    Maker to calculate possible collinear magnetic orderings for a material in VASP.
+
+    Given an input structure, possible magnetic orderings will be enumerated and ranked based
+    on symmetry up to a maximum number of orderings. Each ordering will be
+    relaxed and a higher quality static calculation performed to obtain a total energy.
+    The lowest energy ordering is the predicted ground-state collinear ordering.
+
+    This approach performed decently using VASP for a wide range of test materials in a
+    benchmark. It was originally implemented in atomate (v1) for VASP as the
+    MagneticOrderingsWF.
+
+    Please refer to the following paper for more information and cite appropriately:
+
+        Horton, M.K., Montoya, J.H., Liu, M. et al. High-throughput prediction of the
+        ground-state collinear magnetic order of inorganic materials using Density
+        Functional Theory. npj Comput Mater 5, 64 (2019).
+        https://doi.org/10.1038/s41524-019-0199-7
+
+    .. Note::
+        Good performance of this workflow is ultimately dependent on an appropriate
+        choice of Hubbard U, Hund J values and/or the functional. The defaults will work
+        well for many transition metal oxides.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this Maker.
+    static_maker : StaticMaker
+        Maker used to peform static calculations for total energy.
+    relax_maker : RelaxMaker | None
+        Maker used to perform relaxations of the enumerated structures. If None,
+        relaxations will be skipped (i.e., only static calculations).
+    default_magmoms : dict | None
+        Optional default mapping of magnetic elements to their initial magnetic moments
+        in µB. Generally these are chosen to be high-spin, since they can relax to a
```

Comment:
One thing that might be useful to verify is the default spin on Co. Since the original workflow was written, the pymatgen default changed to low-spin Co, since unlike other elements, Co seemed happier to be trapped in a high-spin configuration even if less stable. Strictly, I think Co should probably be run twice with both configurations.